src/alr/alr-commands.adb

```diff
@@ -143,6 +145,11 @@ package body Alr.Commands is
                      "-c=", "--config=",
                      "Override configuration folder location");
 
+      Define_Switch (Config,
+                     Command_Line_Chdir_Target_Path'Access,
+                     "-C=", "--chdir=",
+                     "Run Alire in the given directory");
```

Comment:
To refer to the command-line tool  proper, better use `alr` here.  (Backticks included as this helps with doc generation later on).

Reply:
10b7716 - thanks!